scripts/plugins/deploy_plugins.py

```diff
@@ -270,28 +270,60 @@ def Deploy_Plugins():
                 )
 
                 try:
-                    result = subprocess.run(
-                        [
-                            plugin_deploy_language,
-                            plugin_deploy_script_path,
-                            stack_action,
-                        ],
-                        universal_newlines=True,
-                        capture_output=True,
+                    # error_file = "stderr_{}.log".format(plugin_name)
+                    # with open(error_file,"wb") as err:
+                    process = subprocess.Popen([
+                        plugin_deploy_language,
+                        plugin_deploy_script_path,
+                        stack_action,
+                    ],
+                    # stdout=subprocess.PIPE,
+                    # TODO: adding any form of stderr to the above call causes the below polling to not work properly for real time output
+                    #       instead, it causes the script to wait until everything is complete before outputting anything
+                    #       tried: stderr=err, stderr=subprocess.PIPE, stderr=subprocess.STDOUT
+                    #       for stderr=err, err is a file: with open(error_file,"wb") as err: 
+                    # stderr=err,
+                    # stderr=subprocess.STDOUT,
+
+                    # TODO: oddly, if you set only stderr=subprocess.PIPE and NOT stdout, 
+                    #       the process.poll() below being focused on stderr does seem to output both stdout and stderr in real time
+                    stderr=subprocess.PIPE,
+                    universal_newlines=True
                     )
 
+                    while True:
+                        # output_stdout = process.stdout.readline()
+                        output_stderr = process.stderr.readline()
+
+                        # if we have a return code, exit the while loop
+                        if process.poll() is not None:
+                            break
+
+                        # if output_stdout:
+                        #     # TODO: parse output for secrets
+                        #     # TODO: specify plugin and output tight output (no extra newlines)
+                        #     # logger.info(output_stdout)
+                        #     print(output_stdout)
```

Comment:
this is probably not necessary, but I left it in because something about the implementation seems off (see previous comment)